xml/http_cookies.xml

```diff
@@ -365,6 +374,8 @@
 
   <fingerprint pattern="^IBMCBR=">
     <description>IBM WebSphere Load Balancer</description>
+    <!-- Replace with a valid example if one is discovered -->
+    <example>IBMCBR=fakevalue</example>
```

Comment:
Could not find a real value for this. Let me know if you find one.

Reply:
I found the format, but not a concrete example in [WebSphere Application Server: Load Balancer Administration Guide - Version 6.0.1](https://public.dhe.ibm.com/software/webserver/appserv/library/v601/infocenter/edge/pdf/LBguide.pdf)

> Active cookie affinity is used to ensure that a client continues to be load balanced
to the same server for some period of time. This is accomplished by sending a
cookie to be stored by the clients browser. The cookie contains the cluster:port:rule
that was used to make the decision, the server that was load balanced to, and a
timeout timestamp for when the affinity is no longer valid. The cookie is in the
following format: IBMCBR=cluster:port:rule+server-time! The cluster:port:rule and
server information are encoded so the CBR configuration is not revealed.

Reply:
Noticed this as well, but I didn't bother trying to write something to conform as to not make the reader think that it actually is a valid value later.

Reply:
No luck finding this when looking at a couple studies vs 80 and 443

Reply:
thx @mkienow-r7 and @tsellers-r7 -- will be keeping this as is.